canonical-web-and-design/snapcraft.io
# Add snap table filter

## Done
Added a filter to the snaps table

## QA
- Go to https://snapcraft-io-3511.demos.haus/admin/ahnuP3quahti9vis8aiw/snaps
- Type in some snap names from the page into the search on the top right
- Check that the table is filtered only showing snaps whose name matches the query

## Issue
Fixes https://github.com/canonical-web-and-design/snap-squad/issues/2032
